cedar-policy/src/api.rs

```diff
@@ -371,6 +386,10 @@ impl Entities {
     ///
     /// Re-computing the transitive closure can be expensive, so it is advised
     /// to not call this method in a loop.
+    /// # Errors
```

Comment:
Here and elsewhere in this file, it might render in the docs html a little nicer (in my opinion/taste) with `## Errors` rather than `# Errors`